app/skins/Main/Main.css

```diff
@@ -1497,7 +1497,7 @@
     -fx-graphic: url("HTMLEditor-Underline.png");
 }
 .html-editor-strike {
-    -fx-graphic: url("HTMLEditor- Strikethrough.png");
+    -fx-graphic: url("HTMLEditor-Strikethrough.png");
```

Comment:
What is this about?

Btw just so you know, due to very bad decision made very long time ago, the Modena skin in not imported in skins, but copied in them, which has devastating effects on maintenance. I will have to go through the entire file, removing the unchanged content.

Reply:
I just removed the whole html-editor css, the one from modena has changed quite a bit and you had copied it unchanged.

Reply:
You do not have to manually go through the whole file btw. Get modena from JDK 8 or whereever you originally copied it from, get a difftool (Meld is a good one) or use IDEA and then when you compare them you will instantly see all unchanged parts ;)

Reply:
That is a good idea. I'll do that in the future